app/classifier/tasks/survey/annotation-view.spec.jsx

```diff
@@ -0,0 +1,80 @@
+import { shallow, mount } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import React from 'react';
+import AnnotationView from './annotation-view';
+
+const annotationValue = {
+  answers: {
+    bat: ['blue'],
+    rat: 'shoe'
+  },
+  choice: 'dog'
+};
+const annotation = {
+  value: [annotationValue]
+};
+
+const onChange = sinon.spy();
```

Comment:
```suggestion
const onChange = sinon.stub().callsFake(annotation => annotation);
```
if you stub this to return the annotation, then you can inspect the annotation with `onChange.returnValues[0]`. That would be useful when you're testing to see if `onChange` is being called with the expected annotation after a delete.